src/sql/workbench/contrib/dataExplorer/browser/dataExplorer.contribution.ts

```diff
@@ -35,13 +36,23 @@ configurationRegistry.registerConfiguration({
 	'title': localize('databaseConnections', "Database Connections"),
 	'type': 'object',
 	'properties': {
-		'datasource.connections': {
-			'description': localize('datasource.connections', "data source connections"),
+		[CONNECTIONS_CONFIG_KEY]: {
+			'description': localize(CONNECTIONS_CONFIG_KEY, "data source connections"),
 			'type': 'array'
 		},
-		'datasource.connectionGroups': {
-			'description': localize('datasource.connectionGroups', "data source groups"),
+		[GROUPS_CONFIG_KEY]: {
+			'description': localize(GROUPS_CONFIG_KEY, "data source groups"),
 			'type': 'array'
+		},
+		[CONNECTION_SORT_BY_CONFIG_KEY]: {
+			'type': 'string',
+			'enum': ['DateAdded', 'DisplayName'],
```

Comment:
I would suggest defining these enum values in the same place as the config keys

```
export const enum ConnectionSortBy {
    dateAdded = 'dateAdded',
    displayName = 'displayName'
}
```

So that you can reference that enum instead of hardcoding the values in the checks above

(and note that the enum values by convention should be camelCased)

Reply:
updated, I also personally prefer this kind of definition over the repeated string literals so I don't need to memorize every place I used same config....